tests/utils/testutils.py

```diff
@@ -257,6 +257,11 @@ def setUpClass(cls):
                 _setup_func_app(TESTS_ROOT / script_dir)
                 cls.webhost = start_webhost(script_dir=script_dir,
                                             stdout=cls.host_stdout)
+            if cls.webhost._proc.errors is not None or \
+               cls.webhost._proc.stderr is not None:
+                raise RuntimeError(
+                    'Failed to start WebHost:'
+                    f'{cls.webhost._proc.errors or cls.webhost._proc.stderr}')
```

Comment:
Wouldn't stderr always be populated? As there could be some processes which would use the STDERR stream?